src/rpcq.lisp

```diff
@@ -533,78 +210,196 @@ We distinguish between the following options for any field type:
     tbl))
 
 
-(defmacro defmessage (class-name field-specs &key (documentation nil))
-  "Create a (de-)serializable message definition with name CLASS-NAME and slots (SLOT-SPECS)."
+(defmacro defmessage (class-name parent-name field-specs &key (documentation nil))
+  "Create a (de-)serializable message definition.  Its class name is CLASS-NAME; its parent message class is named by the list PARENT-NAMES (which is neither () or of length 1); and its slots are specified by FIELD-SPECS.
```

Comment:
I think we need to precisely define what a "parent class" is, esp. if we are going to generate C, which has no classes.

Reply:
I added to the docstring. Let me know if you'd like something more.

Reply:
The docstring you're referencing doesn't exist anymore, this snippet is `Outdated`.

It's a list because (1) I want the syntax to look like `defclass` and (2) no other convention for sending this information to `defmessage` makes any more sense (e.g., `NIL` or a symbol doesn't look any prettier to me).